### ISSUE TITLE
VoiceOver doesn't read text of diff

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#26088]
Testing -186213

Maybe this is related to https://github-com/microsoft/vscode/-/186402, but as I tried it I got different results from that mode but it never seemed to read what I expected

I enable screenreader mode and voiceover, and every time I press F7, this is what it reads. I don't really know what the first part is telling me, and also it says "7 lines changed" but this was a file with 2 change chunks, each with just one changed line, so no idea where it is getting 7.

<img width="662" alt="image" src="https://github-com/microsoft/vscode/assets/323878/fd680884-95e3-409c-9228-2b1486b7cea5">
